css/index.css

```diff
@@ -163,6 +172,8 @@
     grid-template-columns: repeat(2, 1fr);
     grid-template-rows: 1fr;
     gap: 2rem;
+	min-height: auto;
```

Comment:
check the indent, please

Reply:
resolved in https://github.com/innoq/SCS/pull/64/commits/badf6fd2fab28a72921e0b2cc74c25e9c6d72324